src/Services/convert-date.service.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Convert the date from a text to a integer and gives it back as a date string
+ * @param value
+ * @returns {*}
+ * @constructor
+ */
+function ConvertDateService(value) {
```

Comment:
There are probably date parsing and printing methods already implemented in standard libraries. I know this is not part of this PR, but we should do this later on.

Reply:
This file should not be necessary. Maybe also just use another date format in the datepickers.

Reply:
And currently the American date format is used which is confusing. We should use a Swiss / European format.